### ISSUE TITLE
gamemode= does not work

### ISSUE THREAD

**Issue by choke-dev** [ZombieStriker/PsudoCommands#8]
when typing /psudo mute @a[gamemode=spectator] (or any other gamemode) the plugin seems to be unable to execute the command, am i doing something wrong?

**Comment by chrisgdt** [comment]
You're right : argument selector `gamemode=` isn't supported here. If you want to use it, you have to use the 1.12 version `m=spectator`.

I forked the project some months ago and I added all this kind of missing 1.13 feature (+ fixed some other selector that didn't work properly). You can find all informations here : https://github.com/chrisgdt/PsudoCommands
